livecheck/main.py

```diff
@@ -462,15 +490,20 @@ def do_main(*, cat: str, ebuild_version: str, pkg: str, search_dir: Path,
             dn = Path(ebuild).parent
             log.debug('%s -> %s', ebuild, new_filename)
             if settings.keep_old.get(cp, not settings.keep_old_flag):
-                if settings.git_flag:
-                    try:
+                try:
+                    if settings.git_flag:
                         sp.run(('git', 'mv', ebuild, new_filename), check=True)
-                    except sp.CalledProcessError:
-                        log.exception('Error moving %s to %s.', ebuild, new_filename)
-                        return
-                else:
-                    sp.run(('mv', ebuild, new_filename), check=True)
-            Path(new_filename).write_text(content, encoding='utf-8')
+                    else:
+                        copyfile(ebuild, new_filename)
```

Comment:
Sorry I meant to say rename it.

```suggestion
                        ebuild.rename(new_filename)
```

Reply:
Also on line 495 `sp.run(...)` the `ebuild` variable is of type `Path` and it should not be accepted into `sp.run()` arguments where the type there is `Sequence[str]`. Change that line to:

```
                        sp.run(('git', 'mv', str(ebuild), new_filename), check=True)
```

`new_filename` is a `str` already.

Reply:
Fix, Thansk !